cypress/e2e/circulation-log/search.cy.js

```diff
@@ -186,4 +187,22 @@ describe('circulation-log', () => {
     SearchPane.searchByChangedDueDate();
     SearchPane.verifyResultCells();
   });
+
+  it('C16997 Filter circulation log by Claimed returned (firebird)', { tags: [TestTypes.criticalPath, devTeams.firebird] }, () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.visit(TopMenu.usersPath);
+
+    UsersSearchPane.searchByStatus('Active');
+    UsersSearchPane.searchByKeywords(userId);
+    UsersSearchPane.openUser(userId);
+    UsersCard.openLoans();
+    UsersCard.showOpenedLoans();
+    LoansPage.checkAll();
+    LoansPage.claimReturnedAndConfirm('C16997 Filter circulation log by Claimed returned');
+
+    cy.visit(TopMenu.circulationLogPath);
+    SearchPane.searchByClaimedReturned();
+    SearchPane.verifyResultCells();
```

Comment:
What items are check here? All items on page? But we need only items from step LoansPage.claimReturnedAndConfirm
Are we sure that this item were checked?

Reply:
done